_layouts/default.html

```diff
@@ -2,10 +2,16 @@
 <html class="no-js">
   {% include head.html %}
   <body>
-    <!-- Google Tag Manager (noscript) -->
-    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-54RMSDZ"
-    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-    <!-- End Google Tag Manager (noscript) -->
+    
```

Comment:
you don't need to add the Matomo Tag Manager script here because you've already added it to the right place in the head include